projects/swimlane/ngx-ui/src/lib/components/card/card.component.scss

```diff
@@ -0,0 +1,324 @@
+@import 'colors/variables';
+
+$color-accent-grey: linear-gradient(180deg, $color-blue-grey-100 0%, $color-blue-grey-200 100%);
+$card-accent-thickness: 4px;
+
+$color-background-grey: linear-gradient(180deg, $color-blue-grey-750 0%, $color-blue-grey-800 100%);
+$color-background-grey-hover: linear-gradient(180deg, $color-blue-grey-725 0%, $color-blue-grey-775 100%);
+
+$card-horizontal-header-padding: 25px;
+$card-horizontal-gutter: 30px;
+
+$card-outline-border-thickness: 3px;
+
+$card-vertical-header-padding: 15px;
+$card-vertical-gutter: 20px;
+
+@keyframes cardSlideIn {
+  0% {
+    opacity: 0;
+    transform: translateY(40px);
+  }
+
+  100% {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+.ngx-card {
+  display: flex;
+  background: $color-background-grey;
+  box-shadow: 0 0 15px rgba(0, 0, 0, 0.3);
+  border-radius: 2px;
+  cursor: pointer;
+
+  &:hover {
+    background: $color-background-grey-hover;
+  }
+
+  // disabled
+  &.disabled {
+    cursor: default;
+    pointer-events: none;
+
+    &:hover {
+      filter: none;
+    }
+  }
+
+  &--status {
+    display: block;
+    background-color: $color-blue-grey-550;
+    width: 8px;
+    height: 8px;
+    border-radius: 50%;
+    box-shadow: inset 0 -2px 2px $color-blue-grey-600;
+
+    &.success {
+      background-color: #b0e53c;
+      box-shadow: 0 0 7px #b0e53c;
+    }
+
+    &.error {
+      background-color: $color-red-500;
+      box-shadow: 0 0 7px $color-red-500;
+    }
+  }
+
+  &--accent {
+    display: block;
+    background: $color-accent-grey;
+  }
+} // end of ngx-card
+
+.ngx-card-horizontal {
+  position: relative;
+  width: 100%;
+  min-width: 500px;
+  min-height: 80px;
+  height: 80px;
+  transition: all 0.2s ease-in-out;
+  animation: 0.2s ease-in-out cardSlideIn;
+
+  .ngx-card--status {
+    position: absolute;
+    left: 10px;
+    top: 10px;
+  }
+
+  .ngx-card--select {
+    margin-left: $card-horizontal-header-padding;
+
+    .ngx-checkbox--label .ngx-checkbox--box {
+      margin-right: 0;
+    }
+  }
+
+  .ngx-card--accent {
+    position: absolute;
+    width: $card-accent-thickness;
+    min-width: $card-accent-thickness;
+    right: 0;
+    height: 100%;
+    border-radius: 0 2px 2px 0;
+  }
+
+  // horizontal header
+  .ngx-card-header {
+    display: flex;
+    align-items: center;
+    padding: 0 $card-horizontal-header-padding;
+    flex-grow: 1;
+
+    &--title-group {
+      margin-left: $card-horizontal-header-padding;
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      white-space: nowrap;
+
+      .ngx-card-tag {
+        font-weight: bold;
+        font-size: 10px;
+        line-height: 12px;
+      }
+
+      .ngx-card-title {
+        font-weight: 600;
+        font-size: 22px;
+        color: #f0f1f6;
```

Comment:
Question more than an actual comment, do we want to ellipse on long names on vertical card?
![image](https://user-images.githubusercontent.com/62297014/99560083-14239000-298b-11eb-85a8-f1073ec1e633.png)

Reply:
issue: it seems like a fixed height on the card itself (418px) is causing the issue with the footer as shown in Nelson's screenshot with long titles as well

Reply:
Yeah - missed this from design. Long text needs ellipsis.